airgun/widgets.py

```diff
@@ -2238,3 +2240,43 @@ def items(self):
 
     def toggle(self, value):
         self.browser.click(self.ITEM.format(value))
+
+
+class BaseMultiSelect(BaseSelect, Dropdown):
+    """Represents the Patternfly Multi Select.
+
+    https://www.patternfly.org/v4/documentation/react/components/select#multiple
+    """
+
+    BUTTON_LOCATOR = './/button'
+    MENU_BUTTON_LOCATOR = './/button[@aria-label="Options menu"]'
+    OUIA_COMPONENT_TYPE = "PF4/Select"
+
+    def item_select(self, items, close=True):
+        """Opens the Dropdown and selects the desired items.
+
+        Args:
+            items: Items to be selected
+            close: Close the dropdown when finished
+        """
+        if not isinstance(items, (list, tuple, set)):
```

Comment:
Could we handle this if `items` is passed as a string and comma-separated? To split by comma?

Reply:
What you are mentioning is a imperative programming approach, where as I added the an declarative programming approach. The same approach I have seen used in the upstream projects https://github.com/RedHatQE/widgetastic.patternfly4/blob/master/src/widgetastic_patternfly4/select.py#L93 

Let me know your thoughts. Any strong reason to use the imperative programming approach ?

Reply:
I just wanted to add one more validation to the type check.

There is a possibility one will provide string as value to that field.

Since this is not an API to let the user know what field type it is, there are higher chances of misunderstanding the type in UI fields.

Its not a blocking comment though.

Reply:
Added extra support for string class as well in case.